src/generic/c1_plate_helper.cc

```diff
@@ -243,7 +243,7 @@ namespace C1PlateHelper
 /// Mesh pointed to by constraint_mesh_pt.
 //==============================================================================
  void duplicate_corner_nodes(Mesh* bulk_mesh_pt, 
-                             std::map<unsigned,C1CurviLine*> c1_curviline_pt,
+                             const std::map<unsigned,C1CurviLine*>& c1_curviline_pt,
```

Comment:
Passing by const ref rather than the variable (should have done the other arguments whilst I was at it but never mind now)